SeaSharpe_CVGS/SeaSharpe_CVGS/Controllers/OrderController.cs

```diff
@@ -6,8 +6,29 @@
 using System.Net;
 using System.Web;
 using System.Web.Mvc;
-using SeaSharpe_CVGS.Models;
+using SeaSharpe_CVGS.Models;
 
+/*
+ * This Controller tracks orders for site members and employees.
+ * 
+ * **************For Members*********************
+ * Cart:
+ * can add items to a cart
+ * view their cart
+ * delete from their cart
+ * purchase games from their cart
+ * 
+ * OrderHistory
+ * View ordered games and the ship dates
+ * 
+ * **************For Employees*******************
+ * Order Management:
+ * View all completed orders
+ * with partial views for orders waiting to be processed and a partial view to view the selected order
+ * which also provides options to mark the selected order as "processed" as well as print a shipping 
+ * label with the pertinent details.
+ */

```

Comment:
These comments are great, but please move to above:  
public class OrderController : Controller

and if you could switch to /// comments so they get picked up when we print documentation comments.